ocelli-examples/src/main/java/netflix/ocelli/examples/rxnetty/http/RoundRobin.java

```diff
@@ -0,0 +1,91 @@
+package netflix.ocelli.examples.rxnetty.http;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.reactivex.netty.protocol.http.client.HttpClient;
+import io.reactivex.netty.protocol.http.client.events.HttpClientEventsListener;
+import io.reactivex.netty.protocol.http.server.HttpServer;
+import netflix.ocelli.Instance;
+import netflix.ocelli.rxnetty.protocol.http.HttpLoadBalancer;
+import rx.Observable;
+
+import java.net.ConnectException;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+import java.net.SocketException;
+import java.nio.charset.Charset;
+import java.util.concurrent.TimeUnit;
+
+import static io.netty.handler.codec.http.HttpResponseStatus.*;
+
+public final class RoundRobin {
+
+    private RoundRobin() {
+    }
+
+    public static void main(String[] args) {
+
+        Observable<Instance<SocketAddress>> hosts = Observable.just(startServer(OK), startServer(SERVICE_UNAVAILABLE),
+                                                                    new InetSocketAddress(0))
```

Comment:
Do we really need to indent this block so far to the right? :)  It makes the example difficult to read.